.goreleaser.yml

```diff
@@ -6,8 +6,10 @@
 # - Tag a commit with a release, such as 'v0.2.0'
 # - Push the tag
 # - Run `gorelease release` from the root of the repo.
```

Comment:
I noticed this appears to need updating as well when running most recent release. Would you mind fixing here?
```suggestion
# - Run `goreleaser release` from the root of the repo.
```